README.md

```diff
@@ -103,6 +104,12 @@ Python 3.8 or higher is required for running this tool. So, make sure your pytho
    ```
 2. Install JRE/JDK from <a href="https://www.oracle.com/java/technologies/javase/jdk11-archive-downloads.html">oracle</a> if you don't have it installed. Search in google, how to install JDK in windows if you need more guidance on this topic. Recommended jdk version is 11.
 
+### Termux
+
+One Step Installation: Run Below Command
+   ```bash
+   wget -O termux_install.sh https://raw.githubusercontent.com/ratsan/dex2c/termux_install.sh ; chmod +x termux_install.sh ; ./termux_install.sh
```

Comment:
The username has been changed from **ratsan** to **codehasan**. Please modify the link to redirect to this updated repo with the new username.